src/reference-manual/laplace.Rmd

```diff
@@ -0,0 +1,33 @@
+# Laplace Approximation
+
+Stan provides a Laplace approximation algorithm
+which can be used to create an approximate estimate of the variance
+of a distribution given its mode.
```

Comment:
-> which can be used to obtain samples from an approximated posterior

(as the sample transformed from unconstrained to constraint space can be used for mean, variance, quantiles, etc)